README.md

```diff
@@ -58,6 +58,20 @@ Please see: https://github.com/jkyberneees/nodejs-proxy-benchmarks
 Set the base URL for all the forwarded requests. Will be required if `http2` is set to `true`
 Note that _path will be discarded_.
 
+#### queryString
+Set the query string parser and stringifier. By default `fast-proxy` uses 
+[fast-querystring](https://npmjs.com/package/fast-querystring) module. Configuration can be changed like so:
+
+```js
+...
```

Comment:
I believe it's better to provide the full object. Just for sake of clearness.

Reply:
Thanks @RafaelGSS I just updated the pull request